### ISSUE TITLE
Setup OIDC (keycloak) authentication with strapi

### ISSUE THREAD

**Issue by patricksimonian** [bcgov/digital.gov.bc.ca#64]
It looks like the strapi can be initegrated with OIDC (like keycloak) 
> reference https://github.com/strapi/strapi/pull/3161

We will probably need to code up our integration.

## Acceptance Criteria
- [ ] keycloak integrated
- [ ] ideally we do not want keycloak specifics (like host, realm, client) baked into the image this should be configurable
- [ ] document well how the keycloak integration works 
- [ ] a user from a **specific group** within our keycloak realm can access the admin panel via IDIR, others cannot.

**Comment by thegentlemanphysicist** [comment]
Request for a realm has been submitted

**Comment by thegentlemanphysicist** [comment]
Had alignment session with Shelly.  As for who can log in, do we want to allow IDIR BCeID and Github?

**Comment by thegentlemanphysicist** [comment]
Things I learned 3rd party auth does not work on the strapi admin pannel.  The keycloak integrations we saw examples of were being used to allow people to log into the frontend app.  A feature we don't feel is very useful. 

A potential alternative is that we stand up an 'user admin' page that keycloak authorises access to.  like  
https://github.com/nazirov91/ra-strapi-rest
or 
https://github.com/bunred/bunadmin-plugin-strapi-user
This would allow users to edit content, while their access is managed through t 'strapi admin' panel.
